crates/hc_sandbox/src/calls.rs

```diff
@@ -377,6 +387,16 @@ async fn call_inner(cmd: &mut CmdRunner, call: AdminRequestCli) -> anyhow::Resul
             let state = dump_conductor_state(cmd).await?;
             msg!("DUMP CONDUCTOR STATE \n{}", state);
         }
+        AdminRequestCli::DumpNetworkMetrics(args) => {
+            let metrics = dump_network_metrics(cmd, args).await?;
+            // Print without other text so it can be piped
+            print!("{}", metrics);
```

Comment:
Feels like it still might be good to have a newline on the end. Otherwise you might have to flush the pipe if the terminal is not in immediate mode, right?

Reply:
Not sure, it seems to flush before the program ends but maybe I shouldn't be relying on that. Switched the `print` to a `println` :)